pom.xml

```diff
@@ -210,6 +210,14 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>jdk17+</id>
+            <activation>
+                <jdk>[17,)</jdk>
+            </activation>
+            <properties>
```

Comment:
Do we need both this profile and the `jvm.config` file? I'd prefer this one if not.

Also, we could add a todo to remove it once we have upgraded google-fmt to a version that supports 17:

```suggestion
            <properties>
            <!-- TODO: Remove this once google-java-fmt supports 17 -->
```

Reply:
One is for the tests, one is for the actual plugin that formats code in this repo.